python/build-wheel-manylinux2014.sh

```diff
@@ -95,52 +44,12 @@ for PYTHON_NUMPY in "${PYTHON_NUMPYS[@]}" ; do
   # dependency constraints.
   git clean -f -f -x -d -e .whl -e python/ray/dashboard/client -e dashboard/client -e python/ray/jars -e python/requirements_compiled.txt
 
-  (
-    cd python
-    # Fix the numpy version because this will be the oldest numpy version we can
-    # support.
-    /opt/python/"${PYTHON}"/bin/pip install -q numpy=="${NUMPY_VERSION}" cython==0.29.32
-    # Set the commit SHA in __init__.py.
-    if [[ -n "$TRAVIS_COMMIT" ]]; then
-      sed -i.bak "s/{{RAY_COMMIT_SHA}}/$TRAVIS_COMMIT/g" ray/__init__.py && rm ray/__init__.py.bak
-    else
-      echo "TRAVIS_COMMIT variable not set - required to populated ray.__commit__."
-      exit 1
-    fi
-
-    # When building the wheel, we always set RAY_INSTALL_JAVA=0 because we
-    # have already built the Java code above.
-
-    # build ray wheel
-    PATH="/opt/python/${PYTHON}/bin:$PATH" RAY_INSTALL_JAVA=0 \
-    "/opt/python/${PYTHON}/bin/python" setup.py -q bdist_wheel
-
-    # build ray-cpp wheel
-    PATH="/opt/python/${PYTHON}/bin:$PATH" RAY_INSTALL_JAVA=0 \
-    RAY_INSTALL_CPP=1 "/opt/python/${PYTHON}/bin/python" setup.py -q bdist_wheel
-
-    # In the future, run auditwheel here.
-    mv dist/*.whl ../.whl/
-  )
-done
-
-# Rename the wheels so that they can be uploaded to PyPI. TODO(rkn): This is a
-# hack, we should use auditwheel instead.
-for path in .whl/*.whl; do
-  if [[ -f "${path}" ]]; then
-    out="${path//-linux/-manylinux2014}"
-    if [[ "$out" != "$path" ]]; then
-        mv "${path}" "${out}"
-    fi
-  fi
+  ./ci/build/build-manylinux-wheel.sh "${PYTHON}" "${NUMPY_VERSION}"
 done
 
 # Clean the build output so later operations is on a clean directory.
 git clean -f -f -x -d -e .whl -e python/ray/dashboard/client -e python/requirements_compiled.txt
+bazel clean
```

Comment:
without this the part down below can be flaky due to git clean up artifacts and bazel cache doesn't rebuild some artifacts; this is fixed by default in civ2 since we do each build in isolation

Reply:
will this make civ1 wheel building a bit (or a lot) slower?

Reply:
it's hard to tell because there are still the global cache; it might be a lot slower if the global cache has became totally useless; worst case scenario, the part below only takes 5 minutes to run on devbox without a cache 

regardless we'll stop using this script in civ1 after i make civ2 jobs upload to buildkite artifacts